home/init.asm

```diff
@@ -85,19 +130,22 @@ Init:: ; 17d
 	pop af
 	ld [hCGB], a
 
-	call ClearWRAM
+	ClearWRAM
```

Comment:
One-off macros for whole chunks of code is not a style used in the pret disasm projects. If you want to inline these, literally just inline them; replace that `ClearWRAM` macro invocation with the actual code, with blank lines setting it apart as its own "paragraph", and maybe a comment on top like `; clear WRAM`. Likewise for the other two.